test/functional/contested_test.js

```diff
@@ -308,24 +287,20 @@ Scenario('Upload Case Files Confidential Documents @nightly @pipeline', async I
     I.waitForText('Form A Application', '60');
     await I.manualPayment();
     await I.issueApplication();
-    await I.uploadCaseFiles();
-    await I.verifyContestedConfidentialTabData(verifyTabText.historyTab.uploadCaseFiles, verifyTabText.confidentialDocumentsTab);
-    logger.info('Confidential documents verified on Confidential documents tab');
   }
-}).retry(2);
+}).retry(3);
 
-Scenario('Manage Confidential Documents @nightly @pipeline', async I => {
-  if (nightlyTest === 'true') {
+Scenario('Upload Case Files (Confidential Documents) @nightly', async I => {
     //login as a caseworker, create contested case
-    I.signInIdam(caseWorkerUserName, caseWorkerPassword);
-    I.wait('2');
+
+    await I.signInIdam(caseWorkerUserName, caseWorkerPassword);
+    await I.wait('2');
     await I.createCase('FinancialRemedyContested', 'Form A Application');
     await I.contestedCaseworkerCreate(caRef, 'Matrimonial', true);
     await I.contestedDivorceDetails();
     await I.contestedApplicantDetails();
     await I.contestedRespondentDetails();
     await I.contestedNatureOfApplication();
-    await I.contestedOrderForChildren();
```

Comment:
If we select different nature of application then this one is require

Reply:
I have commented all options for in nature of application so  this page will be not displayed. but if want to test this page then I will create separate test where this can be covered.

Reply:
yes please, it would be nice to cover other nature of application options into new tests